### PR TITLE
feat: 2fa via mail authenticator

### PR DESCRIPTION
**Pull request by hill-daniel**

### CODE REVIEW

**Review by webatspeed — APPROVED** [review]

**Review by webatspeed — CHANGES_REQUESTED** [review]
There are conflicting files.

File: src/main/java/de/diakonie/onlineberatung/otp/OtpMailSender.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package de.diakonie.onlineberatung.mail;
+package de.diakonie.onlineberatung.otp;
 
 import de.diakonie.onlineberatung.otp.Otp;
```
Comment (webatspeed) [inline src/main/java/de/diakonie/onlineberatung/otp/OtpMailSender.java]: Not needed anymore, right?

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
+# diakonie-onlineberatung-keycloak-otp
+
+Adds additional endpoints to keycloak to configure 2FA. 
+Currently supports 2FA via:
+* App (e.g. Google Authenticator)
+* Email
+
+## Installation
+* Create a jar (e.g. mvn package)
+* Copy `keycloak-otp-config-spi-<VERSION>-keycloak.jar` into the keycloak deplyoments folder. E.g. `/opt/jboss/keycloak/standalone/deployments`
```
Comment (webatspeed) [inline README.md]: typo in "deplyom..."

**Review by webatspeed — APPROVED** [review]

File: src/main/java/de/diakonie/onlineberatung/otp/OtpMailSender.java (hunk 1 above)
Comment (hill-daniel) [reply src/main/java/de/diakonie/onlineberatung/otp/OtpMailSender.java]: yes :)

**Pull request merged**